gstaichi/python/export_lang.cpp

```diff
@@ -635,21 +635,21 @@ void export_lang(py::module &m) {
 
   py::class_<LaunchContextBuilder>(m, "KernelLaunchContext")
       .def("set_arg_int", &LaunchContextBuilder::set_arg_int)
-      .def("set_arg_int",
+      .def("set_args_int",
```

Comment:
question: is there any reason why we couldn't move the iteration code inside a function, similar to how you are doing for set_args_ndarray?